lib/core/src/lib/form/components/widgets/button/button.widget.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * @license
+ * Copyright © 2005-2025 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable @angular-eslint/component-selector */
+
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import { WidgetComponent } from '../widget.component';
+import { FormService } from '../../../services/form.service';
+import { TranslatePipe } from '@ngx-translate/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatTooltipModule } from '@angular/material/tooltip';
+
+@Component({
+    selector: 'button-widget',
+    templateUrl: './button.widget.html',
+    styleUrl: './button.widget.scss',
+    host: {
+        '[class]': 'hostClasses'
```

Comment:
what about click? maybe we can use @ HostBinding instead of setting it as component metadata?